src/planscape/gis/rasters.py

```diff
@@ -45,17 +49,36 @@ def get_random_output_file(input_file: str, output_folder: str = "/tmp") -> str:
 
 
 def to_planscape(input_file: str) -> List[str]:
-    log.info("converting the raster to planscape format.")
-    warped_output = get_random_output_file(input_file=input_file)
-    cog_output = get_random_output_file(input_file=warped_output)
-
-    warped_raster = warp(
-        input_file=input_file,
-        output_file=warped_output,
-        crs="EPSG:3857",
+    log.info("Converting raster to planscape format.")
+    layer_info = get_layer_info(input_file=input_file)
+    layer_crs = layer_info.get("crs")
```

Comment:
```suggestion
    layer_crs = layer_info.get(settings.CRS)
```
Minor thing. Just to avoid using fixed strings/numbers and follow the same pattern as other variables, like `settings.CRS_FOR_RASTERS`